### DISCUSSION TITLE
Avoid surface "overtagging"

### DISCUSSION THREAD

**Discussion by HalloMike** [streetcomplete/StreetComplete#4645] [Ideas]
Dear all,
i've mapped an designated combined footway- and cycleway with the surface=asphalt because its the same surface for both "lanes".
Meanwhile a StreetComplete User added the additional surface-Tags "cycleway:surface=asphalt" and "footway:surface=asphalt". Result: Now there are 3x surface-Tags in OSM with the same result on a single path.
Based on the OSM-Wiki, the additional surface tags should be used ONLY in case of surface=* is not sufficient.
I'm a mapper in OSM and we should avoid overtagging like this. Especially the OSM newcomers are quickly overstrained with such topics. Keep it simple! :)
How do you think about that?
Best regards
Mike

Example: https://www.openstreetmap.org/way/427224921

**Comment by Helium314** [comment]
This was previously discussed (without change iirc), you might want to read #3222 / #3238

**Reply by HalloMike** [reply]
Thank you very much! I read the discussion/suggestion in #3238, but I have the feeling that the user was not taken seriously. 

SC should follow the rules of OSM, which does not apply in this case. 

Two totally unnecessary tags have been created here by SC. This means an unnecessary extra work for OSM mappers (to re-check the tagging) and - I already mentioned - the topic with the new users.

I would be happy if you could take another look at this.

**Reply by matkoniecz** [reply]
> SC should follow the rules of OSM, which does not apply in this case.

I do not think that any clear OSM rules apply in this case. If you think that some apply, can you link to OSM Wiki page describing them?

> I read the discussion/suggestion in https://github.com/streetcomplete/StreetComplete/issues/3238

Have you seen https://github.com/streetcomplete/StreetComplete/pull/1493#pullrequestreview-266784467 discussion about initial implementation? (including part hidden behind "Load more...")?

**Reply by HalloMike** [reply]
Sure: https://wiki.openstreetmap.org/wiki/Key:cycleway:surface
"...where just surface=* is not sufficient."   &   "...with cycleway with different surfaces"

In my example the surface IS NOT different.

**Reply by Helium314** [reply]
I notice that you omitted "It is useful ..." here, and changed this into " should be used ONLY " in your first post.
This is a considerable difference... please don't do this. Even though I agree with your point that the tagging is unnecessary, such things just trigger some instinctive reaction to oppose you.

**Reply by matkoniecz** [reply]
@HalloMike Well, I am quite familiar with that page (partly because I wrote it, see https://wiki.openstreetmap.org/w/index.php?title=Key:cycleway:surface&action=history )


Also

> where just [surface](https://wiki.openstreetmap.org/wiki/Key:surface)=* is not sufficient

is not making any clear judgment whether such explicit tagging is acceptable/welcome even if it is repetitive (can be interpreted as making to tag it where way has multiple sections, or as being ok to tag only in case of distinct surfaces).

**Reply by HalloMike** [reply]
@Helium314 The reason is that the German translation of this wiki page is more precise than the English one (and my English is not the best, sorry for that).

@matkoniecz: not making clear judgments/fixed rules on a platform used by people all over the world is not a good thing because it leaves room for interpretation.

I mentioned my points above and user apophenic-treehugger reported the same in [#3238](https://github.com/streetcomplete/StreetComplete/issues/3238). The example with maxspeed=60 is also to the point and I wonder here if you really want it that way?

It would be nice if you would take such comments from OSM mappers to heart and I would be happy if you change the logic in SC. If you don't see the need, you can close this ticket.
Thanks

**Reply by matkoniecz** [reply]
> @matkoniecz: not making clear judgments/fixed rules on a platform used by people all over the world is not a good thing because it leaves room for interpretation.

OSM Wiki often states things without going to one side out of compromise between various sides and to describe differing opinions (see sidewalk page for case where it was quite extreme)

> The reason is that the German translation of this wiki page is more precise than the English one

well, if it claims something that is not generally shared by OSM community then it is not really precise (and would not be a correct translation anyway). Not really sure is there clear preference here:

is `surface=asphalt` used for combined asphalt cycleway and paving stones footway acceptable at all?
is `surface=asphalt cycleway:surface=asphalt footway:surface=asphalt` acceptable at all?
is using anything except `surface=paved cycleway:surface=asphalt footway:surface=asphalt` for combined asphalt cycleway and paving stones footway acceptable at all?

> It would be nice if you would take such comments from OSM mappers to heart 

I do, though the part of the complexity is that different people have conflicting tagging opinions and preferences.

> If you don't see the need, you can close this ticket.

It is not an issue, but a discussion (in https://github.com/streetcomplete/StreetComplete/discussions not in https://github.com/streetcomplete/StreetComplete/issues ) and cannot be really closed.

**Comment by mnalis** [comment]
Also, see the Discourse discussion at https://community.openstreetmap.org/t/streetcomplete-and-redundant-surface-tags-at-segregated-paths/5569/6

I agree that extra redundancy is not _perfect_ solution, but two alternatives are (IMHO) much worse, as detailed there:

>While extra redundancy is not perfect, it is a very minor issue compared to much more problematic alternatives, i.e.:
>- **only** the `surface=paving_stones` is tagged: it is unknown if it applies only to footway, or only to cycleway part of the segregated path, or to both. Thus the data consumer may only assume unknown surface.
>- **only** `cycleway:surface=paving_stones` & `footway:surface=paving_stones` are tagged - it is clear what is meant to humans, but most data consumers would only know about `surface=*` and not various potential `*:surface` (and which of them to use anyway), so would show surface as unknown again.

**Reply by mnalis** [reply]
- See also related: 
  - https://github.com/streetcomplete/StreetComplete/issues/4526
  - https://github.com/streetcomplete/StreetComplete/pull/4548
  - https://github.com/streetcomplete/StreetComplete/issues/5878
  - https://github.com/streetcomplete/StreetComplete/issues/3222
  - https://github.com/streetcomplete/StreetComplete/issues/3238

I've created FAQ entry ["Why does StreetComplete add redundant tags like surface?"](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#Why_does_StreetComplete_add_redundant_tags_like_surface?)
